css-env-1/Overview.bs

```diff
@@ -77,6 +77,53 @@ tho user agents may define additional undocumented [=environment variables=]:
 
 * Issue: define list of predefined vars
 
+Safe area inset variables {#safe-area-insets}
+------------------------------------------------------------------
+
+<pre class=propdef>
```

Comment:
I think defining these using `propdef` entries doesn't seem right.  The only aspects of the propdef tables that actually seem to apply here are "Name", "Value", and maybe "Media".  Furthermore, using propdef entries will put them in property indexes, which will be very confusing.

I think in the absence of a better mechanism this should probably be something more like a hand-written table, and if a better mechanism is needed it can be developed later.

Reply:
Fixed